proto/storage/cve.proto

```diff
@@ -114,6 +121,7 @@ message CVEInfo {
   CVSSV2 cvss_v2 = 8;
   CVSSV3 cvss_v3 = 9;
```

Comment:
These fields also appear in message `CVSSScore`.  Are these still relevant, should they be deprecated, are they obsolete due to those other fields?

Reply:
I have added a link in pr description explaining scanner is sending new fields to central https://github.com/stackrox/stackrox/pull/10721/files
We will keep old cvssv2 cvssv3 fields for backward compatibility
But now cvss scores are not just cvssv2 and cvssv3 but a list of CVSS from various sources
Hence we need to store this list